demo/full-featured.md

```diff
@@ -7,6 +7,6 @@ description: This example displays all plugins that come with TinyMCE Cloud's pr
 keywords: example demo custom wysiwyg full-power full-featured
 ---
 
-This example displays all plugins included in the default TinyMCE package, and includes those delivered via TinyMCE Cloud premium subscriptions. Please note, it includes [premium plugins](/pricing/#demo-enterprise).
+This example demonstrates the available TinyMCE plugins, including plugins available on TinyMCE Cloud premium subscriptions (excluding [MoxieManager](/plugins/moxiemanager)). Please note, it includes [premium plugins](/pricing/#demo-enterprise).
 
 {% include codepen.html id="full-featured" height="750" %}
```

Comment:
Just a thought: would it be good to have a paragraph/list of things to try? E.g. "type @ and a name to bring up the Mentions UI" or "do so-and-so to see Tiny Drive in action!"? Personally, I didn't even know what to click to bring up Drive. It took me clicking around for a couple minutes to find the right button...

Reply:
Yeah, that was my first impression of the page. Thinking I'll add a list of "things to try", either before or after the codepen (depending on the size of the list)